jobwatcher/jobwatcher.py

```diff
@@ -51,17 +52,56 @@ def get_asg_name(stack_name, region, proxy_config):
 
     return asg_name
 
-def get_instance_properties(instance_type):
+
+def read_cfnconfig():
+    cfnconfig_params = {}
+    with open(cfnconfig_file) as f:
+        for kvp in f:
+            key, value = kvp.partition('=')[::2]
+            cfnconfig_params[key.strip()] = value.strip()
+    return cfnconfig_params
+
+
+def get_vcpus_from_pricing_file(instance_type):
     with open(pricing_file) as f:
         instances = json.load(f)
         try:
-            slots = int(instances[instance_type]["vcpus"])
-            log.info("Instance %s has %s slots." % (instance_type, slots))
-            return {'slots': slots}
+            vcpus = int(instances[instance_type]["vcpus"])
+            log.info("Instance %s has %s vcpus." % (instance_type, vcpus))
+            return vcpus
         except KeyError as e:
             log.error("Instance %s not found in file %s." % (instance_type, pricing_file))
             exit(1)
 
+
+def get_instance_properties(instance_type):
+    cfnconfig_params = read_cfnconfig()
+    try:
+        cfn_scheduler_slots = cfnconfig_params["cfn_scheduler_slots"]
+        slots = 0
+        vcpus = get_vcpus_from_pricing_file(instance_type)
+
+        if cfn_scheduler_slots is "cores":
```

Comment:
"is" is a test for identity not equality.  This is a bug and will be fixed in the next commit.